django-stubs/db/models/manager.pyi

```diff
@@ -37,7 +37,7 @@ class BaseManager(Generic[_T]):
     # NOTE: The following methods are in common with QuerySet, but note that the use of QuerySet as a return type
     # rather than a self-type (_QS), since Manager's QuerySet-like methods return QuerySets and not Managers.
     def iterator(self, chunk_size: int = ...) -> Iterator[_T]: ...
-    async def aiterator(self, chunk_size: int = ...) -> Iterator[_T]: ...
+    def aiterator(self, chunk_size: int = ...) -> AsyncIterator[_T]: ...
```

Comment:
Weird that the `async` keyword has to be removed as well. But according to this SO post https://stackoverflow.com/a/68911014/177663 that is correct.